modules/nickserv.cpp

```diff
@@ -119,9 +141,18 @@ class CNickServ : public CModule {
             MCString msValues;
             msValues["password"] = GetNV("Password");
             PutIRC(CString::NamedFormat(GetNV("IdentifyCmd"), msValues));
+        } else if(Nick.NickEquals(sNickServName) && sMessage.find("Password accepted") != CString::npos && m_bJoinAfterIdentified) {
```

Comment:
SASL is the alternative.
For this module, I suggest checking how it looks like on various networks which use various services packages, but be prepared for breakage anyway.